### PR TITLE
Add subtitle support for pluzz plugins

### PR DESCRIPTION
**Pull request by melmorabity**
This PR add VOD subtitle support for the pluzz plugin.

### CODE REVIEW

**Comment by beardypig** [comment]
Nice :-D

File: src/streamlink/plugins/pluzz.py [hunk 1]
```diff
@@ -160,7 +176,21 @@ def _get_streams(self):
...
-                yield bitrate, HTTPStream(self.session, video_url)
+                streams.append((bitrate, HTTPStream(self.session, video_url)))
+
+        if self.get_option("mux_subtitles") and videos['subtitles'] != []:
+            substreams = {}
+            for subtitle in videos['subtitles']:
+                # TTML subtitles are available but not supported by FFmpeg
+                if subtitle['format'] == 'ttml':
+                    continue
+                substreams[subtitle['type']] = HTTPStream(self.session, subtitle['url'])
```
Comment (beardypig) [inline src/streamlink/plugins/pluzz.py]: What's in `subtitle['type']`? `MuxedStream` will set whatever the key for the `subtitles` dict to the language of the subtitle.

**Comment by beardypig** [comment]
Looks good, just one question about the language the subtitles will get :)

File: src/streamlink/plugins/pluzz.py (hunk 1 above)
Comment (melmorabity) [inline src/streamlink/plugins/pluzz.py]: subtitle['type'] contains the "kind" of subtitle. The only value I got after checking some videos on Pluzz is 'accessibilité' (for hard of hearing people), in French.

Comment (beardypig) [inline src/streamlink/plugins/pluzz.py]: I suppose that's not too bad. That will show up as the language in VLC/mpv.

**Comment by beardypig** [comment]
Looks good to me. Do you have an geo-unlocked examples with subtitles? :)

**Comment by beardypig** [comment]
Works well mon ami. Going to merge it, thanks a lot :)

**Pull request merged**